app/code/Magento/Sales/Model/Order.php

```diff
@@ -683,10 +683,12 @@ private function canCreditmemoForZeroTotalRefunded($totalRefunded)
         // Case when Adjustment Fee (adjustment_negative) has been used for first creditmemo
         $hasAdjustmentFee = abs($totalRefunded - $this->getAdjustmentNegative()) < .0001;
         $hasActionFlag = $this->getActionFlag(self::ACTION_FLAG_EDIT) === false;
-        if ($isRefundZero || $hasAdjustmentFee || $hasActionFlag) {
+        $creditmemos = ($this->getCreditmemosCollection() === false) ?
+             true : (count($this->getCreditmemosCollection()) > 0);
```

Comment:
Should this be false instead of true? If no credit memo collection was returned that should mean there is currently no credit memo attached to the order and it is not refunded.